go/libraries/doltcore/schema/encoding/schema_marshaling.go

```diff
@@ -151,6 +151,7 @@ type encodedIndex struct {
 	Tags            []uint64 `noms:"tags" json:"tags"`
 	Comment         string   `noms:"comment" json:"comment"`
 	Unique          bool     `noms:"unique" json:"unique"`
+	Spatial         bool     `noms:"spatial,omitempty" json:"spatial,omitempty"`
```

Comment:
you should revert these changes, we don't support spatial indexes in the old format. you should also add some tests to confirm this